lib/SILOptimizer/Mandatory/TFUtilities.cpp

```diff
@@ -631,7 +631,8 @@ tf::createTensorToInt1Inst(SILValue value, SILBuilder &builder,
 /// example) for inlined functions that take and return tensors, since we
 /// know that they are either unreachable or will be inlined into any
 /// clients that use them.
```

Comment:
consider removing this comment block as it's now out of date (the new param is not documented).

(i'd also be fine to keep it in sync with the corresponding comment block in the header file.)

Reply:
Removed the block comment altogether.